packages/react/src/components/tabs/tab-button.tsx

```diff
@@ -15,9 +15,9 @@ interface StyledButtonProps extends IsSelected {
 
 const StyledButton = styled.button<StyledButtonProps>`
     align-items: center;
-    border-bottom: ${({ $isGlobal }) => ($isGlobal ? 'none' : '1px solid #878f9a')}; /* TODO change colors when updating thematization */
+    border-bottom: ${({ $isGlobal, theme }) => ($isGlobal ? 'none' : `1px solid ${theme.component['tab-button-border-bottom-color']}`)}; /* TODO change colors when updating thematization */
     bottom: -1px;
-    color: ${({ $isGlobal }) => ($isGlobal ? '#1B1C1E' : '#878f9a')}; /* TODO change colors when updating thematization */
+    color: ${({ $isGlobal, theme }) => ($isGlobal ? `${theme.component['tab-button-global-color']}` : `${theme.component['tab-button-not-global-color']}`)}; /* TODO change colors when updating thematization */
```

Comment:
Je ne pense pas que 'not-global' c'est un bon pattern de nom, si jamais on a un 3e versions, c'est foutu.

Pour le non global, on pourrait l'enlever 'tab-button-color', ou 'tab-button-normal-color'

Reply:
Ouin. M'a juste l'enlever.